torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1745,7 +1745,9 @@ def produce_guards(
         # DimList[DimConstraint]).  Whenever Optional is accepted, that
         # just means there are no constraints
         constraint_inputs: Optional[InputList[Union[DimConstraint, Optional[DimList[DimConstraint]]]]] = None,
-        _simplified=False
+        _simplified=False,
+        # Indicates if we should produce guards for known static values.
+        ignore_static=True,
```

Comment:
oh we do. For that test, I'd recommend `ignore_static=False`

Reply:
I'll do both :)